tasks/present-or-query.yml

```diff
@@ -30,4 +31,26 @@
   template:
     src: inventory-container-infra.j2
     dest: "{{ os_container_infra_inventory }}"
+
+- name: Add hosts to a dynamic inventory
+  add_host:
+    name: "{{ item.server_name }}"
+    groups:
+      - "cluster"
+      - "{{ os_container_infra_environment_group }}"
+      - "{{ os_container_infra_master_group.name if item.item.name is search('-' + os_container_infra_master_group.name + '$') else omit }}"
```

Comment:
I don't know much about this role, but `item.item.name` looks odd. Is it correct?

Reply:
It's because it's referencing a a previously registered variable that was used in a looped task.